Documentation/Signoffs/Main_Control.md

```diff
@@ -0,0 +1,45 @@
+# Main Control Subsystem
+

```

Comment:
Hi Gabe,

There are arrows leading into this figure that need to have labels so we know what they are.